src/replica/replica_stub.cpp

```diff
@@ -2032,10 +2042,10 @@ task_ptr replica_stub::begin_open_replica(
         if (rep->status() == partition_status::PS_INACTIVE && tsk->cancel(false)) {
             // reopen it
             _closing_replicas.erase(it);
-            _counter_replicas_closing_count->decrement();
+            METRIC_VAR_DECREMENT(closing_replicas);
```

Comment:
If needed, it's better to consider define a new metric type `FunctionGauge`, ref https://github.com/apache/kudu/blob/master/src/kudu/util/metrics.h#L1223

We can improve it in future refactoring.

Reply:
OK, I'll add another issue to track this.